django-reference-app/django-reference-app/static/javascripts/application.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
```

Comment:
We shouldn't check in static assets that from `django-govuk-template`, as they'll be installed when that package is installed.

Also, I _think_ it's still true that the entire contents of the `static` directory should be added to .gitignore, as there's a distinction between the directory where static assets are stored and where they're served from, with the `collectstatic` management command handling copying them to the path they'll be served from - I'd [check the docs](https://docs.djangoproject.com/en/2.0/howto/static-files/) on that though.

Reply:
Spoke with @kerin, this is an issue with the package.  It is fine as is.